gui/default/syncthing/core/syncthingController.js

```diff
@@ -1251,6 +1290,11 @@ angular.module('syncthing.core')
         };
 
         $scope.saveConfig = function (callback) {
+            // set local storage feature
+            localStorage.setItem("showRemoteGUI", $scope.config.gui.showRemoteGUI ? "true" : "false");
```

Comment:
window.localStorage

Reply:
Is there a reason why? Just for curiosity sake because it was working fine without it.

Reply:
Not really a good reason, but I always try to refer to things that are provided by the browser with their full scope name.
Just in case some of some cowboy library we accidentially include that suddenly declares a global localStorage which is not window.localStorage.